docs/commandline.rst

```diff
@@ -90,11 +90,10 @@ each stencil are put in a subfolder named after the stencil:
 The line
 
 .. code-block:: python
-   :caption: my_stencils.gt.py
 
    # [GT] using-dsl: gtscript
 
-is equivalent to writing
+in ``my_stencils.gt.py`` is equivalent to writing
```

Comment:
Not really this PR, but do we want to propose the`.gt.py` ending?

Reply:
I don't know. Probably not.

Reply:
I think it's actually required by the CLI. I vaguely remember that Rico used some `importlib` features using the extension to implement the functionality

Reply:
yes, I also tested it with a file that had just .py suffix.